docusaurus.config.js

```diff
@@ -3,58 +3,114 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
```

Comment:
Why all the changes to this file?

Reply:
I think it's my own VScode prettier formatter that changed this file. 

Maybe we should set up a prettier file to keep the same format everywhere?

Reply:
We could file an issue to do that if you want to file another issue, but in this PR, please revert unnecessary styling changes.